### PR TITLE
Policy improvements

### PR DESCRIPTION
**Pull request by uncomputable**
Makes simplicity::Policy ready for tappy. Depends on #72

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 extern crate bitcoin_hashes;
 extern crate byteorder;
-extern crate miniscript;
+extern crate elements_miniscript as miniscript;
```
Comment (apoelstra) [inline src/lib.rs:19]: In 79387fd1541cf6d067641cace4288a31149bbb52:

If we are going to replace `miniscript` with `elements_miniscript` and keep the name, why did the previous commit in this PR rename `miniscirpt` to `elements_miniscirpt` in a bunch of places?

Also all these `extern crate`s should be pub IMO.

**Comment by apoelstra** [comment]
82c346808a2282f44bd0d647c2fe820c9f1645b5 looks good except the nit about miniscript

File: src/lib.rs (hunk 1 above)
Comment (uncomputable) [reply src/lib.rs]: I changed `elements_miniscript` to `(crate::)miniscript` and replaced `extern crate` with `pub use`.

**Comment by uncomputable** [comment]
Ready for another review

**Review by apoelstra — APPROVED** [review]
ACK 9fd00a79c533a8d25c1a60ac6cc47b8c1830ab64

**Pull request merged**